src/UserSpaceInstrumentation/FindFunctionAddressTest.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/match.h>
+#include <gtest/gtest.h>
+#include <sys/ptrace.h>
+#include <sys/wait.h>
+
+#include "FindFunctionAddress.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/Result.h"
+#include "UserSpaceInstrumentation/Attach.h"
+
+namespace orbit_user_space_instrumentation {
+
+TEST(FindFunctionAddressTest, FindFunctionAddress) {
+  pid_t pid = fork();
+  ASSERT_TRUE(pid != -1);
+  if (pid == 0) {
+    while (true) {
+    }
+  }
+
+  // Stop the child process using our tooling.
+  ASSERT_TRUE(AttachAndStopProcess(pid).has_value());
+
+  auto function_address_or_error = FindFunctionAddress(pid, "printf", "libc.so.6");
+  ASSERT_TRUE(function_address_or_error.has_value());
+
+  function_address_or_error = FindFunctionAddress(pid, "NOT_A_SYMBOL", "libc.so.6");
```

Comment:
another nitpick: looking at this I think that the soname should be before function name.